### PR TITLE
feature/846 async logging

### PR DESCRIPTION
**Pull request by lewisdaly**
Part of #846

- Add CSL_LOG_TRANSPORT env var, default to `file`
- Add event_async_override config for event_sdk enabled charts
- fix some typos

### CODE REVIEW

File: bulk-api-adapter/Chart.lock [hunk 1]
```diff
@@ -0,0 +1,9 @@
+dependencies:
+- name: bulk-api-adapter-service
```
Comment (mdebarros) [inline bulk-api-adapter/Chart.lock]: Please remove

File: bulk-centralledger/Chart.lock [hunk 2]
```diff
@@ -0,0 +1,12 @@
+dependencies:
```
Comment (mdebarros) [inline bulk-centralledger/Chart.lock]: Please remove

File: central/Chart.lock [hunk 3]
```diff
@@ -0,0 +1,12 @@
+dependencies:
```
Comment (mdebarros) [inline central/Chart.lock]: Please remove

File: centraleventprocessor/Chart.lock [hunk 4]
```diff
@@ -0,0 +1,6 @@
+dependencies:
```
Comment (mdebarros) [inline centraleventprocessor/Chart.lock]: Please remove

File: centralkms/Chart.lock [hunk 5]
```diff
@@ -0,0 +1,6 @@
+dependencies:
```
Comment (mdebarros) [inline centralkms/Chart.lock]: Please remove

File: centralledger/Chart.lock [hunk 6]
```diff
@@ -0,0 +1,30 @@
+dependencies:
```
Comment (mdebarros) [inline centralledger/Chart.lock]: Please remove

File: centralledger/chart-service/values.yaml [hunk 7]
```diff
@@ -127,9 +130,11 @@ config:
   kafka_port: 9092
 
   ## Node Configuration
-  log_level: 'info'
+  log_level: info
   ## Enable local logging of events being recorded
   log_events_locally_enabled: false
+
```
Comment (mdebarros) [inline centralledger/chart-service/values.yaml]: Remove line

File: forensicloggingsidecar/Chart.lock [hunk 8]
```diff
@@ -0,0 +1,9 @@
+dependencies:
```
Comment (mdebarros) [inline forensicloggingsidecar/Chart.lock]: Remove

File: ml-api-adapter/Chart.lock [hunk 9]
```diff
@@ -0,0 +1,9 @@
+dependencies:
```
Comment (mdebarros) [inline ml-api-adapter/Chart.lock]: Remove

File: eventstreamprocessor/Chart.lock [hunk 10]
```diff
@@ -0,0 +1,3 @@
+dependencies: []
```
Comment (mdebarros) [inline eventstreamprocessor/Chart.lock]: Remove

File: centralenduserregistry/Chart.lock [hunk 11]
```diff
@@ -0,0 +1,6 @@
+dependencies:
```
Comment (mdebarros) [inline centralenduserregistry/Chart.lock]: Remove

File: monitoring/promfana/Chart.lock [hunk 12]
```diff
@@ -0,0 +1,9 @@
+dependencies:
```
Comment (mdebarros) [inline monitoring/promfana/Chart.lock]: Remove

File: monitoring/efk/Chart.lock [hunk 13]
```diff
@@ -0,0 +1,15 @@
+dependencies:
```
Comment (mdebarros) [inline monitoring/efk/Chart.lock]: Remove

File: mojaloop/Chart.lock [hunk 14]
```diff
@@ -0,0 +1,27 @@
+dependencies:
```
Comment (mdebarros) [inline mojaloop/Chart.lock]: Remove

File: mojaloop-bulk/Chart.lock [hunk 15]
```diff
@@ -0,0 +1,12 @@
+dependencies:
```
Comment (mdebarros) [inline mojaloop-bulk/Chart.lock]: Remove

File: mockpathfinder/Chart.lock [hunk 16]
```diff
@@ -0,0 +1,6 @@
+dependencies:
```
Comment (mdebarros) [inline mockpathfinder/Chart.lock]: Remove

File: centralledger/values.yaml [hunk 17]
```diff
@@ -110,6 +110,9 @@ centralledger-service:
     kafka_host: '$release_name-kafka'
     kafka_port: 9092
 
+    log_level: info
+    log_transport: file
+
```
Comment (mdebarros) [inline centralledger/values.yaml]: No `event_async_override` In this file?

File: mojaloop/values.yaml [hunk 18]
```diff
@@ -112,6 +112,9 @@ central:
         db_create_retry_interval_millis: 200
         db_debug: false
 
+        log_level: info
+        log_transport: file
+
```
Comment (mdebarros) [inline mojaloop/values.yaml]: No `event_async_override` config here?

File: account-lookup-service/Chart.lock [hunk 19]
```diff
@@ -0,0 +1,15 @@
+dependencies:
```
[the remark this answers is not in the document]
Comment (lewisdaly) [reply account-lookup-service/Chart.lock]: I use `git commit -n` now out of habit thanks to `dep:check` ;). But yep, I'll go through and remove them

**Comment by lewisdaly** [comment]
@mdebarros ready to go again

**Pull request merged**